CHANGELOG.md

```diff
@@ -21,6 +20,13 @@ _None_
 
 _None_
 
+## 13.4.0
+
+### New Features
+
+- Add the possibility to configure in `DerivedBuildCodeFormatter` a versioning prefix instead of always defaulting to 1. [#656]
```

Comment:
Just leaving breadcrumbs that while technically this could have been considered a breaking change (due to the change of default value for the prefix), we agreed I  that PR that, since we no there's no current product using this `DerivedBuildCodeFormatter` yet, we're ok with considering this non-breaking and thus to only bump the minor version of the toolkit 👍